Ustring.php

```diff
@@ -220,6 +212,26 @@ public function __construct($string = null)
     }
 
     /**
+     * Check if ext/mbstring is available.
+     *
+     * @return  bool
+     */
+    public static function checkMbString()
+    {
+        return function_exists('mb_substr');
```

Comment:
We don't explicitly promote other projects :-/.
And checking `mb_substr` or any other functions will do its job. The goal is to check that `mbstring` is implemeted :wink:.